server/controllers/finance/vouchers.js

```diff
@@ -115,7 +115,7 @@ function find(options) {
   let typeIds = [];
 
   if (options.type_ids) {
-    typeIds = typeIds.concat(options.type_ids);
+    typeIds = options.type_ids.toString();
```

Comment:
Why was this change necessary?  The previous version escaped an array of values, but this one turns them into a string without escaping.  In theory, I could send `?type_ids=1 OR 1=1` and bypass all filtering via an SQL injection.

Reply:
Because with typeIds = typeIds.concat (options.type_ids); the query filters only the first selected element
![error](https://user-images.githubusercontent.com/8035588/31115713-a11fd32a-a823-11e7-94cd-92e1a90fef88.png)

Reply:
Try using ES6 then - typeIds = [...options.type_ids];

Chances are `type_ids` is no longer being sent as an array, which is what the server API expects.